src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -153,10 +226,466 @@
+ *                     E.g.:<br />
+ *                     <pre>g.chr19:2018023_2018025TTG>GAT</pre><br />
+ *                     This indicates that the bases TTG were changed to GAT from position 2018023 to position 2018025 on chromosome 19.
+ *                 </li>
+ *             </ul>
+ *         </li>
+ *         <li>
+ *             <b><i>annotationTranscript</i></b><br />
+ *             <i>Type: </i>{@link String}<br />
+ *             The ID of the transcript chosen for the detailed Gencode annotation reporting.<br />E.g.: <i>ENST00000435064.1</i><br />
+ *             If the variant allele does not occur within the bounds of any transcript (e.g. is of type <i>IGR</i>), then this field is empty.
+ *         </li>
+ *         <li>
+ *             <b><i>transcriptStrand</i></b><br />
+ *             <i>Type: </i>{@link String}<br />
+ *             The strand direction associated with the transcript on which this variant allele occurs.<br />
+ *             Either "<i>+</i>" or "<i>-</i>".
+ *         </li>
+ *         <li>
+ *             <b><i>transcriptExon</i></b><br />
+ *             <i>Type: </i>{@link Integer} or Empty<br />
+ *             The exon number on the transcript in which this variant allele occurs (1-based).  Corresponds directly to the Gencode exon number.<br />
+ *             If the variant does not occur in the expressed transcript of the corresponding gene (e.g. is of type <i>INTRON</i> or <i>IGR</i>), then this field is empty.
+ *         </li>
+ *         <li>
+ *             <b><i>transcriptPos</i></b><br />
+ *             <i>Type: </i>{@link Integer} or Empty<br />
+ *             Position in the chosen transcript of the variant allele.<br />
+ *             For variant alleles that occur at a single base, the format is simply the position at which that variant occurs in the transcript (1-based, inclusive) (e.g. <i>1294</i>)<br />
+ *             For variant alleles spanning multiple bases, the format is:<br />
+ *             <pre>
+ *             [START]_[END]
+ *             </pre>
+ *             E.g.: <pre>1236_1237</pre><br />
+ *             If the variant does not occur in the expressed transcript of the corresponding gene (e.g. is of type <i>INTRON</i> or <i>IGR</i>), then this field is empty.
+ *         </li>
+ *         <li>
+ *             <b><i>cDnaChange</i></b><br />
+ *             <i>Type: </i>{@link String}<br />
+ *             A {@link String} that summarizes the change resulting from this variant allele in the coding sequence for the transcript in which it occurs.<br />
+ *             Positions in this field are <i>relative to the start of the transcript (1-based, inclusive)</i> unless otherwise noted.<br />
+ *             Generally the format of this field is:<br />
+ *             <pre>
+ *             c.[POSITION][BASES CHANGED]
```

Comment:
Isn't there a dot or underscore between position and bases changed?

Reply:
That's a general description of the field.  The specifics are in the section following that which breaks down the [POSITION] and [BASES CHANGED] placeholders into the actual format.